wordpress-mobile/WordPress-iOS
# Issue/8085 iphonex bg color

Fixes #8085 

To test:

1. Using an iPhone X, launch the app and press the Log In button.
2. Choose any login method and continue until reaching the Login Epilogue screen.
3. Check the background color in the bottom safe area. It should be white, [like this](http://cld.wthms.co/k2V0rV)
